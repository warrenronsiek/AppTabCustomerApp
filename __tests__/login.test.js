/**
 * Created by warren on 5/9/17.
 */

import 'react-native'
import React from 'react'
import Login from '../app/redux/components/login'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('testing login', () => {
  test('renders correctly', () => {
    const login = renderer.create(<Login/>);
    expect(login).toMatchSnapshot()
  })
});

