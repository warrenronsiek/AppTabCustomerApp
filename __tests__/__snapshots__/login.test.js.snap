exports[`testing login renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 2,
        "justifyContent": "center",
        "marginTop": 60,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 40,
          "margin": 10,
          "textAlign": "center",
        }
      }>
      AppTab
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Alpha 0.2.0
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 2,
        "flexDirection": "row",
        "maxHeight": 100,
        "width": 450,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
        }
      }>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              undefined,
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }>
          
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 30,
              },
              undefined,
              Object {
                "fontFamily": "Entypo",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "flex": 5,
          "flexDirection": "column",
          "paddingLeft": 15,
        }
      }>
      <TextInput
        autoCapitalize="none"
        autoCorrect={false}
        defaultValue="email address"
        onChangeText={[Function]}
        style={
          Object {
            "flex": 1,
            "height": 40,
          }
        }
        value={undefined} />
      <TextInput
        autoCapitalize="none"
        autoCorrect={false}
        defaultValue="password"
        onChangeText={[Function]}
        style={
          Object {
            "flex": 1,
            "height": 40,
          }
        }
        value={undefined} />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 4,
        "marginTop": 30,
        "paddingBottom": 120,
      }
    }>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <View
        style={
          Array [
            Object {},
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }>
          Login
        </Text>
      </View>
    </View>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits={
        Array [
          "button",
        ]
      }
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <View
        style={
          Array [
            Object {},
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#0C42FD",
                "fontSize": 18,
                "padding": 8,
                "textAlign": "center",
              },
            ]
          }>
          Register
        </Text>
      </View>
    </View>
  </View>
</View>
`;
